### DISCUSSION TITLE
Add support for changing profile status

### DISCUSSION THREAD

**Discussion by AnalogFeelings** [community/community#136646] [Mobile] [answered]
As said in the title, the mobile app should allow you to change your profile status.

**Comment by eliperkins** [comment, marked as answer]
Hey @AnalogFeelings, thanks for this feedback! Updating user status has shipped on iOS v1.161.0 and will be coming soon on Android. I've added your +1 for this feature to our internal tracking issue for it on Android. Stay tuned!

**Reply by AnalogFeelings** [reply]
I updated GitHub and now I have the feature available. Thanks!

**Reply by eliperkins** [reply]
Awesome! Please keep the feedback coming. ❤️ Folks like you help us improve GitHub Mobile!

**Reply by vanillajonathan** [reply]
It would be great if you could also change the profile picture.
